src/app/notice/page.js

```diff
@@ -114,6 +114,7 @@ export default function Notice() {
               width={138}
               height={92}
               alt="image 498"
+              loading="lazy"
```

Comment:
default is lazy, no need to pass this prop

Reply:
<img width="661" alt="image" src="https://github.com/user-attachments/assets/79b71864-1be5-4dc6-8d0b-60c9003cc0cc">

Reply:
yes sir